models/models.go

```diff
@@ -51,8 +51,9 @@ type Engine interface {
 }
 
 var (
-	x      *xorm.Engine
-	tables []interface{}
+	x                *xorm.Engine
+	supportedDatabse = []string{"mysql", "postgres", "mssql"}
```

Comment:
Small typo nit, should be `supportedDatabase` or `supportedDatabases` and refactored as needed where used.

Reply:
done. It's strange that misspell check failed.